### ISSUE TITLE
[SCREENREADER] Announce zoom in/out context change (FL)

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#17963]
## Issue Description
Veterans using assistive tech to interact with the map need to know what changes have occurred on the map when the zoom buttons have been used. 

---
## Tasks
~- [ ] Determine the amount of change which occurs in % or miles each time the zoom button is clicked~
- [x] Work with Accessibility and designers to identify the best way to announce this change 
- [x] Ensure context changes are communicated, accessible to screen readers.

## Acceptance Criteria
The following should be validated with screen reader and keyboard:
- [x] Each time a Veteran using assistive tech clicks the zoom in button, the screenreader announces ~the level of change which has occurred on the map~ that the map is "zooming in" 
- [x] Each time a Veteran using assistive tech clicks the zoom out button, the screenreader announces ~the level of change which has occurred on the map.~ that the map is "zooming out"
- [x] Add environment check to support testing in staging.

Acceptance criteria has been updated to remove the requirement to specify the amount of zoom based on [this Slack conversation ](https://dsva.slack.com/archives/C0FQSS30V/p1611774857187900)

**Comment by mmiddaugh** [comment]
Closing as unneeded duplicate of #6911

**Comment by mmiddaugh** [comment]
@joshkimux  based on our brief demo during the conversation with the team last week, can this be closed?

**Comment by joshkimux** [comment]
Of course, as demoed, the announcements themselves are looking good to go! Thanks for all the hard work and kudos to @mpelzsherman in particular 😄 🎉

**Comment by mpelzsherman** [comment]
@mmiddaugh To push this to production, we need to remove the environment flag.

**Comment by mpelzsherman** [comment]
Hi @mmiddaugh and @joshkimux . We never pushed this to production because there was no ticket created to remove the environment flag.

I met with @AngelaFowler82 last week and while she was demonstrating using the map with a screen reader, we talked about this feature a bit. She said the current behavior with "zooming in, level X" was not very helpful. She would prefer a zoom radius instead.

**Comment by joshkimux** [comment]
@AngelaFowler82 's recommendation is the way to go 💯 
@mpelzsherman Would we be able to reopen this ticket to edit the behavior to zoom radius instead of levels? I would recommend working with Angela to determine the defect level and priority of this effort.

**Comment by mpelzsherman** [comment]
That would be up to @mmiddaugh and @davidconlon to prioritize. Computing the radius of the map is a bit tricky but it is do-able.

**Comment by mpelzsherman** [comment]
We can use [this chart ](https://docs.mapbox.com/help/glossary/zoom-level/)to compute meters per pixel of the current zoom level, then multiply by the current width of the map (which depends on the screen size) to compute the current search radius.

**Comment by joshkimux** [comment]
@mpelzsherman brilliant stuff! Very curious to see this in action 💯

**Comment by mmiddaugh** [comment]
Captured the most recent feedback in #22649 for grooming/prioritization
